sdk/communication/communication-identity/review/communication-identity.api.md

```diff
@@ -22,9 +22,9 @@ export class CommunicationIdentityClient {
     constructor(endpoint: string, credential: KeyCredential, options?: CommunicationIdentityClientOptions);
     constructor(endpoint: string, credential: TokenCredential, options?: CommunicationIdentityClientOptions);
     createUser(options?: OperationOptions): Promise<CommunicationUserIdentifier>;
-    createUserAndToken(scopes: TokenScope[], options?: OperationOptions): Promise<CommunicationUserToken>;
+    createUserAndToken(scopes: TokenScope[], tokenExpirationInMinutesOrOptions?: number | OperationOptions, options?: OperationOptions): Promise<CommunicationUserToken>;
```

Comment:
Could you provide two overload signatures?
```ts
createUserAndToken(scopes: TokenScope[], options?: OperationOptions): Promise<CommunicationUserToken>;
createUserAndToken(scopes: TokenScope[], tokenExpirationInMinutes: number, options?: OperationOptions): Promise<CommunicationUserToken>;
```
The implementation would remain the same.

Reply:
Done.